packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -974,10 +979,15 @@ export class StatsAnalyzer extends EventsScope {
         result.packetsReceived;
 
       if (this.statsResults[mediaType][sendrecvType].packetsReceived === 0) {
-        LoggerProxy.logger.info(
-          `StatsAnalyzer:index#processInboundRTPResult --> No packets received for ${mediaType} `,
-          this.statsResults[mediaType][sendrecvType].packetsReceived
-        );
+        const receiveSlot = this.receiveSlotCallback(result.ssrc);
+        if (receiveSlot) {
+          LoggerProxy.logger.info(
+            `StatsAnalyzer:index#processInboundRTPResult --> No packets received for receive slot id: "${
```

Comment:
it would be useful to have this logging improvement also in other places where we log issues for receiving video, for example "Packets getting lost from the receiver"  a few lines below

Reply:
Updated.